spec/lib/barcodevalidation_spec.rb

```diff
@@ -47,7 +47,6 @@
     Class.new,
     Object.new,
     0...10,
-    /abcd/,
```

Comment:
why did you remove this?

Reply:
See 49e8f5f

A Ruby regex is not a valid barcode GTIN and it was breaking my factory when using eval, but now that I've changed it to using an array pattern I could revert that.